pkgs/tools/misc/docui/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  name = "docui-${version}";
+  version = "1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "skanehira";
+    repo = "docui";
+    rev = version;
+    sha256 = "1kbap36hccwlj273is98cvgf5z5cl2c3s6p46nh6bnykz3zqzs71";
+  };
+
+  modSha256 = "1qma9bnd4k594cr5dcv74xns53mhfyl4jsm01chf85dxywjjd9vd";
+
+  meta = with stdenv.lib; {
+    description = "TUI Client for Docker";
+    homepage = https://github.com/skanehira/docui;
+    license = licenses.mit;
+    platforms = platforms.linux;
```

Comment:
Does this even build on Darwin? Darwin does not ship with Docker obviously, but it does have docker support through other means. If this compiles on it, we should open it up to Darwin as well.

Reply:
I don't have a darwin machine available, but it should work in theory. 
I added darwin platform, let's give bot a go at building?

Reply:
sure. @GrahamcOfBorg build docui